### PR TITLE
chore(curriculum): remove `__locals` from tower of hanoi puzzle project

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64de1469fccce61940f498c4.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ The `rods` dictionary will represent the three rods with their disks. Give it th
...
 
 ```
 
 `rods['A']` should be an empty list.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    a = __locals.get("rods")
+    a = rods
     a['A'] == []
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64de1469fccce61940f498c4.md:26]:
```suggestion
    rods['A'] == []
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64de1469fccce61940f498c4.md [hunk 2]
```diff
@@ -31,15 +31,15 @@ Your `rods` dictionary should have an `'A'` key.
...
 
 ```
 
 `rods['B']` should be an empty list.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    a = __locals.get("rods")
+    a = rods
     a['B'] == []
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64de1469fccce61940f498c4.md:43]:
```suggestion
    rods['B'] == []
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64de1469fccce61940f498c4.md [hunk 3]
```diff
@@ -48,15 +48,15 @@ Your `rods` dictionary should have a `'B'` key.
...
 
 ```
 
 `rods['C']` should be an empty list.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    a = __locals.get("rods")
+    a = rods
     a['C'] == []
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64de1469fccce61940f498c4.md:60]:
```suggestion
    rods['C'] == []
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/657b6a4a49faec5c600287ba.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ You should reduce the indentation level of all the code after the `return` state
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    hanoi = __locals.get("move")
+    hanoi = move
     a, b, c = [3, 2, 1], [], []
     hanoi(3, a, b, c)
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/657b6a4a49faec5c600287ba.md:22]:
```suggestion
    a, b, c = [3, 2, 1], [], []
    move(3, a, b, c)
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
image

Looks like quite a few instances of `__locals` still.

**Comment by ilenia-magoni** [comment]
Thank you, now I should have got all of them!

**Review by Dario-DC — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
No locals found in this block

**Pull request merged**